apis/apiextensions/v1/composition_transforms.go

```diff
@@ -172,10 +174,14 @@ func (m MapTransform) MarshalJSON() ([]byte, error) {
 func (m *MapTransform) Resolve(input any) (any, error) {
 	switch i := input.(type) {
 	case string:
-		val, ok := m.Pairs[i]
+		raw, ok := m.Pairs[i]
```

Comment:
```suggestion
		p, ok := m.Pairs[i]
```

Nit, just because `raw.Raw` below stutters.

Reply:
Good catch. Thanks.